flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/LaunchableMesosWorker.java

```diff
@@ -220,6 +220,7 @@ public String toString() {
 		taskInfo.addAllResources(allocation.takeScalar("cpus", taskRequest.getCPUs(), roles));
 		taskInfo.addAllResources(allocation.takeScalar("gpus", taskRequest.getGPUs(), roles));
 		taskInfo.addAllResources(allocation.takeScalar("mem", taskRequest.getMemory(), roles));
+		taskInfo.addAllResources(allocation.takeScalar("disk", taskRequest.getDisk(), roles));
```

Comment:
What is the default Mesos value if `disk` is not specified?

Reply:
Hello Till,

Sorry for the delay in the response.
I tested flink on the mesos cluster we have at Criteo and before my change it didn't work because it was taking the default 0 value for the disk limit.
In our cluster we enforce the disk usage check but setting this parameter --enforce_container_disk_quota.
You can find the description of the parameter here: http://mesos.apache.org/documentation/latest/configuration/agent/
(and here http://mesos.apache.org/documentation/latest/configuration/agent/ - http://mesos.apache.org/documentation/latest/isolators/disk-du/)
The default installation of mesos has false for enforce_container_disk_quota so the disk limit is never checked. 

After my change, it works correctly on our mesos cluster (if we set a value different from the default 0 value using the new parameter).

Reply:
Hmm, looking at the Mesos source code it seems that the disk size will be automatically calculated if not specified. Thus, I would suggest to only set the `"disk"` resource value if the value is larger than `0.0`.

Reply:
sure, I'll do the change :)